core/src/main/resources/hudson/model/User/sidepanel.jelly

```diff
@@ -30,8 +30,16 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
   <l:header />
   <l:side-panel>
+    <div class="jenkins-app-bar">
+      <div class="jenkins-app-bar__content jenkins-build-caption">
+        <l:icon src="${h.getUserAvatar(it, '96x96')}" class="jenkins-avatar" />
+        <h1>
+          ${it.fullName}
+        </h1>
+      </div>
+    </div>
```

Comment:
Why is there no Jelly tag for this?

Reply:
Good point, added support for icons in App bars.